bundle/tests/bundle/loader.go

```diff
@@ -0,0 +1,26 @@
+package bundle
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/bundle/config/mutator"
+	"github.com/stretchr/testify/require"
+)
+
+func loadTarget(t *testing.T, path, env string) *bundle.Bundle {
+	ctx := context.Background()
+	b, err := bundle.Load(ctx, path)
+	require.NoError(t, err)
+	err = bundle.Apply(ctx, b, bundle.Seq(mutator.DefaultMutatorsForTarget(env)...))
+	require.NoError(t, err)
+	err = bundle.Apply(ctx, b, bundle.Seq(
+		mutator.RewriteSyncPaths(),
+		mutator.MergeJobClusters(),
+		mutator.MergeJobTasks(),
+		mutator.MergePipelineClusters(),
+	))
+	require.NoError(t, err)
+	return b
+}
```

Comment:
Note: this is a copy from `bundle/tests/loader.go` to use it from the glob test. Follow up: move the glob test to the parent package so that it can use the reference loader.